pkg/remote/builtin/github/github.go

```diff
@@ -34,12 +37,14 @@ type GitHub struct {
 
 func New(conf *config.Config) *GitHub {
```

Comment:
I think the data model is 97% locked down. There may be some minor tweaks, but overall I'm pretty confident. So yes, we can definitely start integrating GitLab.

We'll probably want to get this PR merged first, since it modifies the `Remote` interface and has some changes that you'll need